Vagrantfile

```diff
@@ -80,9 +80,8 @@ Vagrant.configure(2) do |config|
     winw1.winrm.password = "vagrant"
 
     if not File.file?(".lock/joined") then
-     # Update containerd
-     puts "calico: #{calico_version}; containerd: #{containerd_version}"
```

Comment:
Is there a reason to remove this?

Reply:
@aravindhp This like has not been removed, but replaced with line 87 

```
puts "[Vagrantfile] Provisioning winw1 node with Calico: #{calico_version}; containerd: #{containerd_version}"
```

However, `puts` are being removed in the `master-windows-native` branch, see https://github.com/kubernetes-sigs/sig-windows-dev-tools/commit/0e2b4ce4af9690ae0fb802e1af5606078d03975c, as those from-Vagrantfile logs are inconsistent and ambiguous and even annoying.

1. Annoying, those are displayed for every `vagrant` command, even if irrelevant, e.g. `vagrant ssh` against existing machine. And, if we run `vagrant` 20x via `mage`, then those `puts` unnecessarily pollute the lengthy output of the whole workflow.

2. Ambiguous because logged values depend on which YAML is loaded and that depends on how `vagrant` was executed. For a visual example:
      
      ```
      cp variables.yaml variables.local.yaml
      # edit variables.local.yaml replacing containerd 1.6 with 1.7
      mage                                # displays my new 1.7
      vagrant ssh controlplane   # displays 1.6 version
      ```
      or      
      ```
      VAGRANT_VARIABLES=my.yaml vagrant up   # my.yaml requests version  3.25.1
      vagrant ssh controlplane                                # now puts displays version 3.25.0
      ```
            
      So, instead of sprinkling puts in `Vagrantfile`, logging of settings is realised by Magefiles, e.g. https://github.com/kubernetes-sigs/sig-windows-dev-tools/blob/master-windows-native/magefiles/main.go#L104-L109  By the way, I'm now also working on improving and validating `variables.yaml`, to catch failures earlier than after 20 minutes of `vagrant up`, etc.

TL;TR: We should aim to remove any need to run `vagrant` command directly, but via `mage` only. The only command one may want or need to run directly is `vagrant ssh` and then those `puts` logs are irrelevant.